server/src/main/java/org/elasticsearch/search/aggregations/bucket/histogram/LongBounds.java

```diff
@@ -218,6 +215,16 @@ public Long getMax() {
         return max;
     }
 
+    public boolean contain(long time) {
```

Comment:
`time` seems like a confusing name for this parameter.  Maybe just `value` like the double version uses?